app.js

```diff
@@ -0,0 +1,25 @@
+var express = require('express');
+var app = express();
+
+app.get('/api/v1/accounts/:id', (req, res) => {
```

Comment:
This route is just supposed to return account information. not specific transactions.